lib/msf/base/sessions/mysql.rb

```diff
@@ -8,6 +8,8 @@ class Msf::Sessions::MySQL < Msf::Sessions::Sql
   # @param [Hash] opts
   def initialize(rstream, opts = {})
     @client = opts.fetch(:client)
+    self.platform = opts.fetch(:platform, nil)
+    self.arch = opts.fetch(:arch, nil)
```

Comment:
```suggestion
    self.platform = opts.fetch(:platform)
    self.arch = opts.fetch(:arch)
```

Reply:
Context: The value won't be optional, as we want the caller to always supply a value here. Using `opts.fetch(:arch, nil)` is just the same as `opts[:arch]`, which doesn't give us any runtime guarantees that a key has been set.